Examples/README-Google.md

```diff
@@ -0,0 +1,38 @@
+# Using AppAuth for iOS and macOS with Google
+
+To configure the sample with a Google OAuth client, first visit
```

Comment:
This page is also excellent for quickly configuring an iOS client:

https://developers.google.com/mobile/add?platform=ios